src/query/QueryManager.js

```diff
@@ -59,4 +68,50 @@ export default class QueryManager {
 
     this.scene.requestRender();
   }
+
+  highlightSelectedArea(geometry) {
+    this.unhiglight();
+    if (!geometry) return;
+    const coordinates = geometry.coordinates;
+    if (geometry.type === 'MultiPolygon') {
+      const entity = new Entity();
+      coordinates[0].forEach(coords => {
+        const convertedCoords = coords.map(c => {
+          const degCoords = lv95ToDegrees(c);
+          return Cartesian3.fromDegrees(degCoords[0], degCoords[1]);
+        });
+
+        const ent = new Entity({
+          polygon: {
+            hierarchy: convertedCoords,
+            material: OBJECT_HIGHLIGHT_COLOR.withAlpha(0.7)
+          }
+        });
+        entity.merge(ent);
+      });
+      this.viewer.entities.add(entity);
+      this.highlightEntity = entity;
+    } else if (geometry.type === 'MultiLineString') {
+      const convertedCoords = coordinates[0].map(c => {
+        const degCoords = lv95ToDegrees(c);
+        return Cartesian3.fromDegrees(degCoords[0], degCoords[1]);
+
+      });
+      this.highlightEntity = this.viewer.entities.add({
+        polyline: {
+          positions: convertedCoords,
+          material: OBJECT_HIGHLIGHT_COLOR
+        }
+      });
+    }
+    this.scene.requestRender();
+  }
```

Comment:
For other geometries please do a console.error so that we can easily detect and fix the issue.